theseus/utils/examples/motion_planning/motion_planner.py

```diff
@@ -11,7 +11,7 @@
 import theseus as th
 
 
-class MotionPlanner:
+class MotionPlannerObjective(th.Objective):
```

Comment:
Nice this does look more streamlined. We can potentially do this for other example later.

Now we are starting to really have something like downloadable pre-trained models that can be then further customized!